src/apis/adapters_api.ts

```diff
@@ -16,7 +16,7 @@ import {
 import {
     SimpleInterestTermsContractContract,
     CollateralizedSimpleInterestTermsContractContract,
-} from "src/wrappers";
+} from "../wrappers";
```

Comment:
RIP absolute paths?

Reply:
Can't seem to make these work in the build process.   I've gone deep down this rabbit hole and don't want to do it again 😬